app/pages/share/AccessManagement.js

```diff
@@ -623,6 +636,42 @@ export const AccessManagement = (props) => {
           </Button>
         </DialogActions>
       </Dialog>
+      <Dialog
+        id="resendInvite"
+        aria-labelledBy="dialog-title"
+        open={showResendInviteDialog}
+        onClose={() => setShowResendInviteDialog(false)}
+      >
+        <DialogTitle onClose={() => setShowResendInviteDialog(false)}>
+          <MediumTitle id="dialog-title">{t('Confirm Resending Invitation')}</MediumTitle>
+        </DialogTitle>
+        <DialogContent>
+          <Body1>
+            {t(
+              'You invited {{inviteName}} to view your data on {{inviteDate}}. Are you sure you want to resend this invitation?',
+              {
+                inviteName: selectedSharedAccount?.name || selectedSharedAccount?.email,
+                inviteDate: formattedInviteDate,
+              }
+            )}
+          </Body1>
```

Comment:
The only thing I notice here is that the Figma design shows the `inviteName` and `inviteDate` in bold text, so it may be worth using the `<Trans />` component here to allow that.  

However, if you do go ahead with this, we should likely bold the email in the "Revoke Invitation" dialog as well to match it.  Up to you.

I also feel it may read a little better if the "Are you sure you want to resend this invitation?" part is on a new line (i.e. in separate `Body1` element), but again, that's more of a question of design intent in the Figma file (i.e. it's on a new line there, but is that the intent, or just coincidental).